module/Decision/view/decision/decision/search.phtml

```diff
@@ -36,13 +36,15 @@ $this->headTitle($this->translate('Search for decision')); ?>
 
         <?php if (isset($result)): ?>
             <ul>
-                <?php foreach ($result
-
-                as $decision): ?>
-                <li><?= $decision->getMeeting()->getType() ?> <?= $decision->getMeeting()->getNumber() ?>
-                    .<?= $decision->getPoint() ?>
-                    .<?= $decision->getNumber() ?> <?= $decision->getContent() ?>
-                    <?php endforeach ?>
+                <?php foreach ($result as $decision): ?>
+                    <?php $type = $decision->getMeeting()->getType() ?>
+                    <?php $number = $decision->getMeeting()->getNumber() ?>
+                    <li><a href="<?= $this->url('decision/meeting', ['type' => $type, 'number' => $number]) ?>">
+                            <?= $type ?> <?= $number ?>
+                            .<?= $decision->getPoint() ?>
+                            .<?= $decision->getNumber() ?>
+                        </a> <?= $decision->getContent() ?>
```

Comment:
Don't forget to add a closing `<li>`

Reply:
And while you are at it, could you format the decision number something like this:
        
    implode('.', [$number, $decision->getPoint(), $decision->getNumber()]);

These space in the number really hurt my eyes...